listenbrainz/mbid_mapping/reports/top_discoveries.py

```diff
@@ -0,0 +1,135 @@
+from datetime import datetime, timezone
+
+import psycopg2
+import psycopg2.extras
+from psycopg2.errors import OperationalError
+
+from mapping.utils import insert_rows, log
```

Comment:
Not sure if this import will work, may need to modify it to `..mapping.utils`.

Reply:
It has worked so far...